package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
- [ ] Kindly fix this error
![image](https://github.com/ridabensalem/hello-rails-react/assets/24703171/e06e3ce6-8b41-447b-abf8-14abd0c8f6b5)
I get this error when I run `npm run build` to build your project before running it.